admin_app/src/app/dashboard/page.tsx

```diff
@@ -52,50 +52,56 @@ const Dashboard: React.FC = () => {
   }, []);
 
   return (
-    <>
-      <Box sx={{ display: "flex", marginTop: 4, flexDirection: "row" }}>
-        <ClickAwayListener onClickAway={() => setSideBarOpen(false)}>
-          <Sidebar
-            open={sideBarOpen}
-            setOpen={setSideBarOpen}
-            setDashboardPage={setDashboardPage}
-            selectedDashboardPage={dashboardPage}
-          />
-        </ClickAwayListener>
+    <Box
+      sx={{
+        display: "flex",
+        marginTop: 4,
+        flexDirection: "row",
+        minWidth: "900px",
+        maxWidth: "1500px",
```

Comment:
This line adds a gray space on my laptop. Can we remove the `maxWidth` or make it 1900px? That seems to render well on my laptop

<img width="1688" alt="image" src="https://github.com/user-attachments/assets/68b1ae45-4a82-491a-aa34-256772b4f3a4">

Reply:
Increased to 1900. Adding grey space was my intended behaviour. Should the dashboard page itself be centered do you think? Sidebar must stick to the left but contents of the dashboard could be centered... @Tanmay-97 @suzinyou